app/script/entity/Conversation.js

```diff
@@ -122,15 +122,15 @@ z.entity.Conversation = class Conversation {
 
       const knownNotificationStates = Object.values(z.conversation.NotificationSetting.STATE);
       if (knownNotificationStates.includes(this.mutedState())) {
-        const isStateOnlyMentions = this.mutedState() === z.conversation.NotificationSetting.STATE.ONLY_MENTIONS;
-        const isInvalidState = isStateOnlyMentions && !this.selfUser().inTeam();
+        const isExpectedState = this.mutedState() === z.conversation.NotificationSetting.STATE.MENTIONS_AND_REPLIES;
```

Comment:
I don't quite understand this renaming. Why is the `mention_and_replies` the `expectedState`?